team_picker/public/js/assing_teams.js

```diff
@@ -0,0 +1,31 @@
+function myFunction() {
```

Comment:
please don't name a function like this. a function is an action so it should start with a verb.